src/sentry/static/sentry/app/views/issueList/filters.tsx

```diff
@@ -2,52 +2,44 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
 
+import {Organization, SavedSearch} from 'app/types';
 import {PageHeader} from 'app/styles/organization';
 import {trackAnalyticsEvent} from 'app/utils/analytics';
 import {t} from 'app/locale';
 import PageHeading from 'app/components/pageHeading';
 import QueryCount from 'app/components/queryCount';
-import SentryTypes from 'app/sentryTypes';
 
 import IssueListSearchBar from './searchBar';
 import IssueListSortOptions from './sortOptions';
 import SavedSearchSelector from './savedSearchSelector';
 
-class IssueListFilters extends React.Component {
-  static propTypes = {
-    organization: SentryTypes.Organization,
-
-    savedSearchList: PropTypes.arrayOf(SentryTypes.SavedSearch),
-    savedSearch: SentryTypes.SavedSearch,
-
-    sort: PropTypes.string,
-    query: PropTypes.string,
-    isSearchDisabled: PropTypes.bool,
-    queryCount: PropTypes.number,
-    queryMaxCount: PropTypes.number,
-
-    onSortChange: PropTypes.func,
-    onSearch: PropTypes.func,
-    onSidebarToggle: PropTypes.func,
-    onSavedSearchSelect: PropTypes.func.isRequired,
-    onSavedSearchDelete: PropTypes.func.isRequired,
-    tagValueLoader: PropTypes.func.isRequired,
-    tags: PropTypes.object.isRequired,
-  };
+type Props = {
+  organization: Organization;
+
+  savedSearchList: SavedSearch[];
+  savedSearch: SavedSearch;
+
+  sort: string;
+  query: string;
+  isSearchDisabled: boolean;
+  queryCount: number;
+  queryMaxCount: number;
 
+  onSortChange: (sort: string) => void;
+  onSearch: (query: string) => void;
+  onSidebarToggle: (event: React.MouseEvent) => void;
+  onSavedSearchSelect: (search: SavedSearch) => void;
+  onSavedSearchDelete: (search: SavedSearch) => void;
+  tagValueLoader: React.ComponentProps<typeof IssueListSearchBar>['tagValueLoader'];
```

Comment:
maybe you could create:  `type IssueListSearchBarProps = React.ComponentProps<typeof IssueListSearchBar>`

and reuse it?